data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3360 @@
+		"Generator (Furnace Class)"
+		"Korath Repeater Rifle" 20
+		"Pulse Rifle" 50
+		"Dual Pulse Pistols" 10
+	gun 0 -7 "Korath Detainer"
+	gun -22 -102 "Sunbeam"
+	gun 22 -102 "Sunbeam"
+	gun -34 -68 "Sunbeam"
+	gun 34 -68 "Sunbeam"
+	gun -49 -38
+	gun 49 -38
+	gun -61 -14 "Flamethrower"
+	gun 61 -14 "Flamethrower"
+	turret -51 21.5 "Korath Grab-Strike"
+	turret 51 21.5 "Bullfrog Anti-Missile"
+	turret -56.5 75 "Korath Banisher"
+	turret 56.5 75 "Bullfrog Anti-Missile"
+	turret 0 85.5 "Bullfrog Anti-Missile"
+
+mission "Unfettered: Start Of The War"
+	name `Strength Of A Leader`
+	description `Prove your strength against Si-Ra and Hi-Ye in a battle. But do not kill them.`
+	source "Darkcloak"
+	to offer
+		has "event: Unfettered: shield beetle mass production"
+		has "event: Unfettered: new cooling system"
+		has "Unfettered: Special Cargo 2: done"
+		has "main plot completed"
+		has "language: Wanderer"
+	on offer
+		conversation
+			`You find Si-Ra and Hi-Ye waiting for you on the spaceport. Si-Ra starts speaking to you, "Are you ready for war, <last>?"`
+			choice
+				`	"Not yet, I will come back when I am."`
+					goto later
+				`	"I was born ready."`
+			`	"And may you die ready, as well." Hi-Ye comes closer and takes you aside to speak to you, privately. "I have managed to buy myself a ship with the money from our last business opportunity. It is much stronger than the old one! And as a warlord, he has a strong fleet as well. You may want to consider this seriously. Do not forget you can call the fight off, to continue it later once you are ready, by giving us a symbolic fee."`
+			choice
+				`	"You do not scare me."`
+					accept
+				`	"After thinking a bit... Maybe I should not take such risks, I will not participate in your war."`
+					decline
+			`	"That is good! Let us fight, on the same terms of honor as before. Take off when you will be ready, we will be waiting for you."`
+			label later
+			`	"Be sure to come back soon, war cannot wait forever."`
+				defer
+	
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		ship "Shield Beetle (Pulse)" "Intrepid II"
+		ship "Shield Beetle Prototype (Black)" "Tejolis"
+		conversation
+			action
+				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 100
+				event "shield beetles accessible"
+			`	"You sure know how to fight, but sheer firepower may not be enough, you should still train more before we are ready to start the war. Also we will require more jump drives than what we currently have, because our Alpha supplies have been disrupted by your actions."`
+	
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on complete
+		log `Officialy joined the Unfettered Militia. It should be the last time I have to prove my worth to them. At the very least I may now buy all of their ships with this license.`
+		"salary: Unfettered" = 5000
+		set "license: True Unfettered"
+		event "joined the unfettered"
+		"reputation: Wanderer" <?= -1000
```

Comment:
```suggestion
		#"reputation: Wanderer" <?= -1000
```
I'm thinking this should probably be commented out until a2 is at least also ready for testing PR as well, and that PR can uncomment it?

As is you don't actually join the Unfettered in all out war against the Wanderers yet, but still end up locked out of progressing the rest of the story from the only angle available.

Reply:
Yeah... idk if I can justify that. You get the Unfettered color. With the Hai it can be negociated but with the Wanderers they know what UHai do now, they raid them
For testing purpose yeah but when this pr will be merged I'll still have to uncomment it I think.